core/src/main/java/com/linecorp/armeria/common/logging/NoopRequestLogBuilder.java

```diff
@@ -68,6 +69,9 @@ public void requestHeaders(HttpHeaders requestHeaders) {}
     @Override
     public void requestContent(@Nullable Object requestContent, @Nullable Object rawRequestContent) {}
 
+    @Override
+    public void onRequestContent(HttpData data) {}
```

Comment:
I'm curious if we should rename `onRequest/ResponseContent` and `increaseRequest/ResponseLength` to the same name, because they basically do something similar without much difference. What do you think?

Reply:
I think it's a great idea. What would be good for the new name?

Reply:
I'm not sure if it's great but how about just `increaseRequest/ResponseLength`? :sweat_smile: e.g.

```java
logBuilder.increaseRequestLength(data);
// or if uninterested in feeding the previewer, probably in a unit test?
logBuilder.increaseRequestLength(length);
```